pkg/services/alerting/conditions/query.go

```diff
@@ -221,8 +220,8 @@ func (c *QueryCondition) executeQuery(context *alerting.EvalContext, timeRange l
 			for _, frame := range frames {
 				ss, err := FrameToSeriesSlice(frame)
 				if err != nil {
-					return nil, errutil.Wrapf(err,
-						`request handler failed to convert dataframe "%v" to plugins.DataTimeSeriesSlice`, frame.Name)
+					return nil, fmt.Errorf(
+						`request handler failed to convert dataframe "%v:" to plugins.DataTimeSeriesSlice: %w`, frame.Name, err)
```

Comment:
looks like you don't need the first colon here.  guessing from a search and replace

Reply:
good catch! i'll remove. thanks for reviewing!